plugins/modules/grafana_folder.py

```diff
@@ -281,6 +314,9 @@ def main():
             folder = grafana_iface.get_folder(title)
             changed = True
         folder = grafana_iface.get_folder(title)
+        if permissions is not None:
```

Comment:
That won’t be idempotent

Reply:
Compare permissions from module and API in main. Remove not needed items in `get_folder_permission`